From a1eadadf586f888bd397bd3d4e405dc8a93bde57 Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Tue, 22 Sep 2020 05:05:24 -0700
Subject: [PATCH 1621/1852] drm: xlnx: bridge: Fix UNUSED_VALUE type warning

commit a15c6a3cc6f9ce6cb7b80b9c0138a63b4aca6d1b from
https://github.com/Xilinx/linux-xlnx.git

This patch fix UNUSED_VALUE type coverity warning.

warning:
Assigning value "-22" to "ret" here, but that stored value is
overwritten before it can be used.

Overwriting previous write to "ret" with value from
"strncpy_from_user(cmd, buf, size)".

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_bridge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_bridge.c b/drivers/gpu/drm/xlnx/xlnx_bridge.c
index 6ee462ada676..120902cf3f49 100644
--- a/drivers/gpu/drm/xlnx/xlnx_bridge.c
+++ b/drivers/gpu/drm/xlnx/xlnx_bridge.c
@@ -344,7 +344,7 @@ static ssize_t xlnx_bridge_debugfs_write(struct file *f, const char __user *buf,
 		char *cmd, **tmp;
 		char *w, *h, *f;
 		u32 width, height, fmt;
-		int ret = -EINVAL;
+		int ret;
 
 		cmd = kzalloc(size, GFP_KERNEL);
 		ret = strncpy_from_user(cmd, buf, size);
-- 
2.31.1

